layouts/community/ergodox/osx_neo2/keymap.c

```diff
@@ -472,15 +476,26 @@ bool process_record_user_shifted(uint16_t keycode, keyrecord_t *record) {
         SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_6) SS_UP(X_LALT));
         break;
       case NEO2_3:
-        // There is no OSX key combination for the script small l character
+        if (command) {
+          SEND_STRING(SS_DOWN(X_LGUI) SS_DOWN(X_LSHIFT) SS_TAP(X_3) SS_UP(X_LSHIFT) SS_UP(X_LGUI));
```

Comment:
These can be improved with eg.

```suggestion
          tap_code16(S(G(KC_3)));
```

Saves some bytes and cycles.

Reply:
Thanks for the tap_code() suggestion, saves ~860 bytes in final size (yay!).